### ISSUE TITLE
Vim paste empty

### ISSUE THREAD

**Issue by kwuiee** [wfxr/minimap.vim#23]
## Check list

- [x] I have read through the [README](https://github.com/wfxr/minimap.vim/blob/master/README.md) (especially F.A.Q section)
- [x] I have searched through the existing issues

## Environment info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Others:

## Version info

```shell
vim --version
VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Jun  1 2020 20:24:08)
Included patches: 1-884
Compiled by conda@64a8cedf8525
```

## Problem / Steps to reproduce

File `a.txt`

```
I am minimap
```

File `b.txt`

```
Who are you?
```

In file a.txt, `vim a.txt` and click `yy` in `MinimapClose` mode. In file b.txt, `vim b.txt` and click `p` in `Minimap` mode, only get `\n` pasted, as below

```
Who are you?

```

**Comment by wfxr** [comment]
Should be fixed by #24 .

@kwuiee  Thanks for your feedback!

**Comment by kwuiee** [comment]
> 
> 
> Should be fixed by #24 .
> 
> @kwuiee Thanks for your feedback!

It's fixed now. Appreciate your help! :smile:
